html-block-and-inline/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Block and Inline</title>
+  </head>
+  <body>
+    <h1>Blue American Snuggie&reg; Blanket</h1>
+    <h2>by SnuggieStore</h2>
+    <p>Price: <s>$19.99</s> $14.99</p>
+    <p>
+      Part of our new American collection, this <em>beautiful</em> blue is
+      already a customer favorite!
+    </p>
+    <p>Enjoy complete <b>Freedom of Movement!</b></p>
```

Comment:
Instead of a `<b>` tag it's more semantically correct to use `<strong>` in this case.